extensions/wikia/Chat2/Chat.class.php

```diff
@@ -58,8 +54,9 @@ public static function echoCookies() {
 		if ( !$wg->User->isLoggedIn() ) {
 			return [ "key" => false ];
 		}
-		$key = "Chat::cookies::" . sha1( $wg->User->getId() . "_" . microtime() . '_' .  mt_rand() );
-		$wg->Memc->set( $key, [ "user_id" => $wg->User->getId(), "cookie" => $_COOKIE ] , 60 * 60 * 48 );
+		$key = "Chat::cookies::" . sha1( $wg->User->getId() . "_" . microtime() . '_' . mt_rand() );
+		$wg->Memc->set( $key, [ "user_id" => $wg->User->getId(), "cookie" => $_COOKIE ], 60 * 60 * 48 );
```

Comment:
this looks really messy - what about moving `60 * 60 * 48` and `Chat::cookies::` to consts, maybe some `implode` with `_`  and each param of `set()` can be moved to a separate line.
What would you say about even moving the key creation to a separate function?